### PR TITLE
Add CONTRIBUTORS file

### LINKED ISSUE #7: Add CONTRIBUTORS file

### PR DESCRIPTION
**Pull request by IlanZuckerman**
fix #7

### CODE REVIEW

**Comment by lioramilbaum** [comment]
@IlanZuckerman Press the re-review button if your PR is ready for my review (again)

**Comment by IlanZuckerman** [comment]
fix #7

**Comment by IlanZuckerman** [comment]
@lioramilbaum thanks for reviewing. I dont understand, what else need to be addressed?
i changes the first message to "fix #7" and changed the topic of this PR to "Add CONTRIBUTORS file"

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
 # baboon
+Ilan test2
```
Comment (lioramilbaum) [inline README.md]: This is the third comment. Why did you change the README file?

File: README.md [hunk 2]
```diff
@@ -1 +0,0 @@
-# baboon
```
Comment (lioramilbaum) [inline README.md]: why have you removed this line? This file should not be in this PR at all

**Comment by lioramilbaum** [comment]
We will cover the PR in the session today and fix together whatever needed to be fixed.
